settings.py

```diff
@@ -282,3 +284,11 @@
 MME_SLACK_ALERT_NOTIFICATION_CHANNEL = 'matchmaker_alerts'
 MME_SLACK_MATCH_NOTIFICATION_CHANNEL = 'matchmaker_matches'
 MME_SLACK_SEQR_MATCH_NOTIFICATION_CHANNEL = 'matchmaker_seqr_match'
+
+#########################################################
+#  AnVIL Terra API specific settings
+#########################################################
+
+TERRA_API_CONFIG = {
```

Comment:
I think it would be easier if you just used top-level varianble for this, so 
```
TERRA_API_URL = 'https://api.firecloud.org/'
```

Reply:
I was expecting there were some more configurations. If there are no more settings, I will flatten it to the top-level.